src/reaper_osara.cpp

```diff
@@ -331,6 +331,31 @@ bool isItemSelected(MediaItem* item) {
 	return *(bool*)GetSetMediaItemInfo(item, "B_UISEL", NULL);
 }
 
+char* automationModeAsString(int mode) {
+	// this works for track automation mode and global automation override.
+	switch (mode) {
+	case -1:
+		return "no override";
```

Comment:
Consider making this string shorter. Right now, this is going to result in an announcement like "Global automation override no override". I wonder if we could just say "none" here?

Reply:
Changed to "none"